auth/auth_user.py

```diff
@@ -31,7 +36,11 @@ def logout_page(cls,email,username,allocation):
 			except AttributeError:
 				pass
 		else:
-			allocation_html = "<p>Cloud Status: <span style=color:red>•</span> Disconnected</p>"
+			if connect:
```

Comment:
if allocation.status == AllocationCloudEnum.{some property that means it's connecting}